pkg/controller/pravegacluster/upgrade.go

```diff
@@ -111,10 +121,61 @@ func (r *ReconcilePravegaCluster) clearUpgradeStatus(p *pravegav1alpha1.PravegaC
 	return nil
 }
 
-func (r *ReconcilePravegaCluster) syncComponentsVersion(p *pravegav1alpha1.PravegaCluster) (err error) {
-	var synced bool
+func (r *ReconcilePravegaCluster) rollbackClusterVersion(p *pravegav1alpha1.PravegaCluster, version string) (err error) {
```

Comment:
Say the operator crashes and restarts in the middle of a triggered rollback. What happens in this case? Are we able to pick up from where the previous instance left off?

Reply:
Yes. We would be able to pick up from where we left off last.